uisdk/src/test/java/com/karhoo/uisdk/screen/booking/domain/quotes/KarhooAvailabilityTest.kt

```diff
@@ -4,16 +4,13 @@ import androidx.arch.core.executor.testing.InstantTaskExecutorRule
 import androidx.lifecycle.LifecycleOwner
 import androidx.lifecycle.LifecycleRegistry
 import androidx.lifecycle.Observer
-import com.karhoo.sdk.api.model.LocationInfo
-import com.karhoo.sdk.api.model.Quote
-import com.karhoo.sdk.api.model.QuoteId
-import com.karhoo.sdk.api.model.QuoteList
-import com.karhoo.sdk.api.model.QuoteVehicle
+import com.karhoo.sdk.api.model.*
```

Comment:
It's the default AS behavior. Detekt doesn't complain about this. 
I find unnecessary having the imports listed one-by-one instead of importing all if all are used